### DISCUSSION TITLE
COE Starter Toolkit - Premium connectors not displaying, only standard connectors

### DISCUSSION THREAD

**Discussion by gcusworth** [microsoft/powerapps-tools#1019] [Q&A]
The COE starter kit has only detected standard connectors e.g SharePoint however premium connectors such as Dataverse are being used on several apps and flows. Is this a known issue and are there any specific flows I could check please? The app detail Power BI report only shows standard connectors for apps, not premium connectors. Thanks

**Comment by manuelap-msft** [comment]
Please raise issues under the Issues tab https://github.com/microsoft/powerapps-tools/issues where, once raised, they will be automatically assigned to someone on the CoE team. The discussions tab should be for questions only!
